onnxruntime/core/framework/environment.cc

```diff
@@ -57,6 +57,22 @@ Status Environment::Initialize() {
             "Constrain input and output types to float tensors.")
         .TypeAndShapeInferenceFunction(propagateShapeAndTypeFromFirstInput);
 
+    ORT_ATTRIBUTE_UNUSED ONNX_OPERATOR_SCHEMA(ScaledTanh)
```

Comment:
ScaledTanh was an experimental op which was removed from onnx... You should add this to contrib defs to make sure older version models can still work... Please follow this PR https://github.com/Microsoft/onnxruntime/pull/588

Reply:
But why MeanVarianceNormalization is here?

Reply:
MeanVarianceNormalization got promoted into a regular opschema. but it doesnt have a historical version stored in ONNX (because it was an experimental op before. putting it here is just for winml I think).